mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -490,7 +490,7 @@ class LogisticRegression @Since("1.2.0") (
 
   protected[spark] def train(
       dataset: Dataset[_],
-      handlePersistence: Boolean): LogisticRegressionModel = {
+      handlePersistence: Boolean): LogisticRegressionModel = Instrumentation.instrumented { instr =>
```

Comment:
To avoid line too wide, we might want to import `instrumented` and save "Instrumentation" from this line.